### ISSUE TITLE
about reproduce performance

### ISSUE THREAD

**Issue by wanboyang** [YuanEZhou/Grounded-Image-Captioning#2]
Thanks for opening source code.
The provided pretrained model can perform the results claimed in your paper, but when I fail to training model on flicker dataset to reproduce the results.

**Comment by wanboyang** [comment]
I use the default setting and features provided in this project. The result on flicker test split is 
![scan_flicker](https://user-images.githubusercontent.com/23290289/97083915-fd1ca700-1645-11eb-95ff-aad9b89c8197.png)
the SCST training starts about 30000 iterations, it is interesting to find the cider scores drops and always below 0.60

**Comment by wanboyang** [comment]
should I do something to reproduce the results performed by pretrained models?

**Comment by YuanEZhou** [comment]
It seems that you just do the first stage traning(cross entropy loss), you should continue to do the second stage traning(self critical loss).

**Comment by wanboyang** [comment]
![QQ图片20201026141506](https://user-images.githubusercontent.com/23290289/97139525-ac1bc880-1795-11eb-9748-d06ac754183c.png)
I achieved whole training processing stage, the CIDEr always below 0.62

**Comment by YuanEZhou** [comment]
Strange!  Maybe some hyperparameters were not set correctly in the cleaned version.  I will check it later.

**Comment by YuanEZhou** [comment]
Sorry for the late reply.  For better performance on CIDEr, you should comment [line12](https://github.com/YuanEZhou/Grounded-Image-Captioning/blob/77295a6e36de817f173435e809effc3396469ee3/misc/SCAN/ground_reward.py#L12)  and uncomment [line 11](https://github.com/YuanEZhou/Grounded-Image-Captioning/blob/77295a6e36de817f173435e809effc3396469ee3/misc/SCAN/ground_reward.py#L11).

**Comment by wanboyang** [comment]
Thanks for replying
